bin/common/start_docker_service.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+sed -i 's/DOCKER_OPTS="\(.*\)"/DOCKER_OPTS="\1 --storage-driver=overlay2"/' /etc/sysconfig/docker
```

Comment:
I probably took it out because the `--host $IP` flag was added, but I think you mentioned later that we might not need the docker engine to bind to an external interface anymore (I never modified it to do that).